build.md

```diff
@@ -0,0 +1,80 @@
+# Building your project's front-end
+
+We have a standard. Please stick to it.
```

Comment:
Ha, yes it is. I meant to remove it from the original before putting it here and forgot.